glean-core/ios/Glean/Glean.swift

```diff
@@ -67,7 +67,7 @@ public class Glean {
 
         self.handle = withFfiConfiguration(
             dataDir: getDocumentsDirectory().absoluteString,
-            packageName: Bundle.main.bundleIdentifier!,
+            packageName: AppInfo.name,
```

Comment:
Mh, I think this should be the `bundeldentifier`, as far as I can tell [from here](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html)

Reply:
Yeah, I needed to commit it so I can build the sample application and see what the values actually are.

Reply:
Oh wait, I just moved `Bundle.main.bundleIdentifier!` into the `AppInfo` struct. All good here.